src/earn/EarnHome.test.tsx

```diff
@@ -159,16 +160,21 @@ describe('EarnHome', () => {
         <MockedNavigator
           component={EarnHome}
           params={{
-            activeEarnTab: EarnTabType.OpenPools,
+            activeEarnTab: EarnTabType.AllPools,
           }}
         />
       </Provider>
     )
 
+    // All Pools
     expect(
       queryByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
-    ).toBeFalsy()
+    ).toBeTruthy()
+    expect(
+      getByTestId('PoolCard/arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216')
+    ).toBeTruthy()
     fireEvent.press(getByText('earnFlow.poolFilters.myPools'))
+    // My Pools
```

Comment:
Maybe should also assert here that we don't see the Eth pool

Reply:
Added in 737593bcfa64e72ec22cea78c982fbbf3b03ee94!